open_in_mpv/open_in_mpv.py

```diff
@@ -193,6 +193,10 @@ def real_main(log: TextIO) -> int:
         logger.exception('No URL was given')
         print(json.dumps(dict(message='Missing URL!')))
         return 1
+    if 'https' not in url:
```

Comment:
There should be a setting.

Reply:
yeah the idea is to integrate this into the extension so the user can decide if they wish to load non secure urls. but by default open-in-mpv should disallow, have to have a json response from the extension telling open-in-mpv to disregard this check.